po/hr.po

```diff
@@ -0,0 +1,279 @@
+msgstr "Pulse Efekti"
+
+#: data/com.github.wwmm.pulseeffects.appdata.xml.in:8
+#: data/com.github.wwmm.pulseeffects.desktop.in:5
+msgid "Audio Effects for PulseAudio Applications"
+msgstr "Zvučni efekti za Pulseaudio aplikacije"
+
+#: data/com.github.wwmm.pulseeffects.appdata.xml.in:9
+msgid "Wellington Wallace"
+msgstr "Wellington Wallace"
+
+#: data/com.github.wwmm.pulseeffects.appdata.xml.in:11
+msgid ""
+"PulseEffects is an advanced audio manipulation tools. It includes an "
+"equalizer, limiter, compressor and a reverberation tool, just to mention a "
+"few. To complement this there is also a built in spectrum analyzer."
+msgstr ""
+"Pulse Efekti je napredan alat za manipulaciju zvukom. Uključuje ekvalizator, "
+"graničnik, kompresor i alat odjeka, i druge alate. Još sadrži i ugrađeni "
+"analizator spektra."
+
+#: data/com.github.wwmm.pulseeffects.appdata.xml.in:16
+msgid ""
+"Because PulseEffects uses the default PulseAudio sound server it will work "
+"with most, if not all, applications you use. All supported applications are "
+"presented in the main window, where each can be enabled individually."
+msgstr ""
+"Zato jer Pulse Efekti koriste zadani PulseAudio zvučni poslužitelj radit će "
+"s većinom, ako ne i sa svim aplikacijama koje koristite. Sve podržane "
+"aplikacije su prikazane u glavnom prozoru, gdje se svaka pojedinačno može "
+"omogućiti."
+
+#: data/com.github.wwmm.pulseeffects.appdata.xml.in:22
+msgid ""
+"Besides manipulating sound output, PulseEffects is able to apply effects to "
+"an input device, such as a microphone. This is, for example, useful in audio "
+"recording, but it also works well during voice conversations."
+msgstr ""
+"Osim manipuliranja izlaza zvuka, Pulse Efekti mogu primijeniti efekte i na "
+"ulazni uređaj, poput mikrofona. To je na primjer, korisno pri snimanju "
+"zvuka, ali isto tako radi i tijekom razgovora."
+
+#: data/com.github.wwmm.pulseeffects.appdata.xml.in:27
+msgid ""
+"When PulseEffects is launched it will conveniently remember the "
+"configuration used in the last session. It is also possible to save all the "
+"current settings as profiles."
+msgstr ""
+"Kada je Pulse Efekti pokrenut, upotrijebit će podešavanja od prošlog puta. "
+"Još je moguće spremiti sve trenutne postavke kao predloške."
+
+#: data/com.github.wwmm.pulseeffects.appdata.xml.in:40
+msgid "Equalizer - Input"
+msgstr "Ekvalizator - Ulaz"
+
+#: data/com.github.wwmm.pulseeffects.appdata.xml.in:44
+msgid "Equalizer - Output"
+msgstr "Ekvalizator - Izlaz"
+
+#: data/com.github.wwmm.pulseeffects.appdata.xml.in:56
+msgid "Calibration"
+msgstr "Kalibracija"
+
+#: data/com.github.wwmm.pulseeffects.desktop.in:4
+msgid "Audio Enhancer"
+msgstr "Zvučni poboljšivač"
+
+#: data/com.github.wwmm.pulseeffects.desktop.in:6
+msgid "limiter;compressor;reverberation;equalizer;autovolume;"
+msgstr "graničnik;kompresor;odjek;ekvalizator;automatska glasnoća zvuka;"
+
+#: data/com.github.wwmm.pulseeffects.desktop.in:9
+msgid "pulseeffects"
```

Comment:
Just an heads up, that last "pulseeffects" string is the icon name and must not be translated. Currently I don't know how to exclude this. We might have to do something similar to [this](https://github.com/gnome-mpv/gnome-mpv/blob/master/data/io.github.GnomeMpv.desktop.in#L6)